packages/p2p-v2/src/pages/my-profile/screens/MyProfileDailyLimit/MyProfileDailyLimit.tsx

```diff
@@ -2,30 +2,25 @@ import React, { useState } from 'react';
 import { Button } from '@deriv-com/ui/dist/components/Button';
 import { Text } from '@deriv-com/ui/dist/components/Text';
 import DailyLimitModal from '../../../../components/Modals/DailyLimitModal/DailyLimitModal';
-import { useDevice } from '../../../../hooks';
+import { useAdvertiserStats, useDevice } from '../../../../hooks';
 import './MyProfileDailyLimit.scss';
 
-type TMyProfileDailyLimitProps = {
-    buyLimit: number;
-    currency: string;
-    sellLimit: number;
-};
-
-const MyProfileDailyLimit = ({ buyLimit, currency, sellLimit }: TMyProfileDailyLimitProps) => {
+const MyProfileDailyLimit = () => {
     const [isModalOpen, setIsModalOpen] = useState(false);
     const { isMobile } = useDevice();
+    const { data: advertiserStats } = useAdvertiserStats();
 
     return (
         <>
             <div className='p2p-v2-my-profile-daily-limit'>
                 <Text color='less-prominent' lineHeight='sm' size='xs'>
                     Want to increase your daily limits to{' '}
                     <Text color='less-prominent' lineHeight='sm' size='xs' weight='bold'>
-                        {buyLimit} {currency}{' '}
+                        {advertiserStats?.daily_buy_limit} USD{' '}
```

Comment:
@nada-deriv I retrieved the data from hook, `useAdvertiserStats` transforms the data from `useAdvertiserInfo`